src/mapping.ts

```diff
@@ -59,6 +63,66 @@ export function handleCreateProxy(event: Created): void {
   proxy.save()
 }
 
+function loadOrCreatePool(poolMeta: PoolMeta, block: EthereumBlock): Pool {
+  let pool = Pool.load(poolMeta.id)
+
+  log.debug("pool start block {}, current block {}", [poolMeta.startBlock.toString(), block.number.toString()])
+  if (pool == null && parseFloat(block.number.toString()) >= poolMeta.startBlock) {
+    createPool(poolMeta.id.toString())
+    pool = Pool.load(poolMeta.id)
+  }
+
+  log.debug("successfully using this for pool meta id: {}", [poolMeta.id.toString()])
+  return <Pool>pool    
+}
+
+function createYesterdaySnapshot(date: BigInt, block: EthereumBlock): void {
+  let yesterdayTimeStamp = date.minus(BigInt.fromI32(86400))
```

Comment:
I think it's better to explicitly declare variables like this, e.g. `let dayInSeconds = 86400; ` or add a comment to the line explaining that it's one day.

Reply:
good call